client/cmdhfmf.c

```diff
@@ -1530,6 +1557,423 @@ static int CmdHF14AMfNestedHard(const char *Cmd) {
     return 0;
 }
 
+
+static int CmdHF14AMfHardAuto(const char *Cmd) {
+    /*
+        Author: Matthias Konrath
+        Company: Trustworks GmbH
+        Email: m.konrath@trustworks.at
+    */
```

Comment:
you get your fame in the changelog and in the commit message..
or/and you can add a line on the top of this file.